internal/resources/templates/bids_results.go

```diff
@@ -22,6 +22,8 @@ const BidsResults = `
 			<div class="ui tabs divider"></div>
 		</div>
 		<div class="ui container">
+		<div class="ui grid">
+		<div class="column" style="width:65%">
```

Comment:
Switch the order of "content" and "history" `div` and change the width to 80% content and 20% history.